libs/src/sdk/types/File.ts

```diff
@@ -0,0 +1,8 @@
+// Type representing a file in Node environment
+export type NodeFile = {
+  buffer: Buffer
+  name: string
+}
+
+// Type representing a file in Node and browser environments
+export type CrossPlatformFile = NodeFile | File
```

Comment:
I think this is fine, esp because FIle is more convenient in browser. but if we wanted to keep things consistent we could also use the buffer+name object in browser right?

Reply:
Ah actually node `Buffer` and js `ArrayBuffer` aren't quite compatible so we would still need to differentiate, think I'm gonna leave it as is for now

Reply:
gotcha gotcha rip